### PR TITLE
Add option to not sort

### LINKED ISSUE #615: ls -U equivalent for lsd?

### PR DESCRIPTION
**Pull request by MichaelAug**
Add ls -U equivalent flag

closes #615 

#### TODO

- [X] Use `cargo fmt`
- [X] Add necessary tests
- [X] Add changelog entry

### CODE REVIEW

File: doc/lsd.md [hunk 1]
```diff
@@ -122,6 +122,9 @@ lsd is a ls command with a lot of pretty colours and some other stuff to enrich
 `--sort <WORD>...`
 : Sort by WORD instead of name [possible values: size, time, version, extension]
 
+`-U`, `--unsorted`
+: Do not sort
```
Comment (meain) [inline doc/lsd.md:126]: Just to keep it consistent with gnu ls.

```suggestion
: Do not sort. List entries in directory order
```

**Review by meain — CHANGES_REQUESTED** [review]
Looks good, just have a few tiny nitpicks.

**Comment by MichaelAug** [comment]
Thanks for the feedback! All comments addressed and changelog updated :)

**Comment by meain** [comment]
LGTM. Thanks for working on this :D

**Pull request merged**